### PR TITLE
Player 2320

### PR DESCRIPTION
**Pull request by atopilskii**

### CODE REVIEW

File: VRSampleApp/app/src/main/java/com/ooyala/sample/screen/VideoFragment.java [hunk 1]
```diff
@@ -83,6 +84,17 @@ public void onResume() {
     if (player != null) {
       player.resume();
     }
+    if (null != playerController) {
```
Comment (IvanSakharovskii) [inline VRSampleApp/app/src/main/java/com/ooyala/sample/screen/VideoFragment.java]: 2 different codestyles in 4 lines :)

File: VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoFragment.kt [hunk 2]
```diff
@@ -71,14 +72,28 @@ open class VideoFragment() : Fragment(), Observer {
...
     player?.resume()
+    if(playerController != playerController) {
+      playerController.onResume(activity, this)
+    }
   }
 
   override fun onPause() {
     super.onPause()
     player?.suspend()
+    if(playerController != playerController) {
```
Comment (IvanSakharovskii) [inline VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoFragment.kt]: What is this check?

**Review by AlinaVoronkovaEpam — APPROVED** [review]

File: VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoFragment.kt [hunk 3]
```diff
@@ -71,14 +72,22 @@ open class VideoFragment() : Fragment(), Observer {
     }
   }
 
+  override fun onDestroy() {
+    super.onDestroy()
+    playerController.onDestroy()
```
Comment (IvanSakharovskii) [inline VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoFragment.kt:76]: add null-check for kotlin

Comment (atopilskii) [reply VRSampleAppKotlin/app/src/main/java/com/ooyala/sample/screen/VideoFragment.kt:76]: it isn't necessary

**Review by IvanSakharovskii — APPROVED** [review]

**Pull request merged**